<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScene</class>
 <widget class="QMainWindow" name="MainScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainScene</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="soundMenu">
    <property name="title">
     <string>音效：开</string>
    </property>
    <addaction name="actionopen_2"/>
    <addaction name="actionclose_2"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="soundMenu"/>
  </widget>
  <action name="actionexit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionopen_2">
   <property name="text">
    <string>开</string>
   </property>
  </action>
  <action name="actionclose_2">
   <property name="text">
    <string>关</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
